lib/core/utils/ui_constants/keys.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+const Key gridViewBuilderKey =  Key("Grid view builder with data on snapshot");
```

Comment:
Keys should be lower snake case, and try to have more shortest naming of the keys

Reply:
Done